README.md

```diff
@@ -1,94 +1,23 @@
-# Open Source Checklist
+# cfgov-setup
 
-Automating CFPB's [Open Source Checklist](https://github.com/cfpb/open-source-project-template/blob/master/opensource-checklist.md) for easy release of internal source code.
+This package provides a central home for our logic for building front-end assets as part of the Python build process (for example, when generating a [wheel](https://pypi.python.org/pypi/wheel)), which has until now simply been duplicated everywhere. Open source examples:
 
-![](https://raw.githubusercontent.com/cfpb/open-source-checklist/master/screenshot.png)
+https://github.com/cfpb/complaint/blob/v1.2.3/setup.py#L20
+https://github.com/cfpb/retirement/blob/0.5.0/setup.py#L20
```

Comment:
Tiny-winey thing. Maybe these should be a list. They end up as a single line [when rendered](https://github.com/rosskarchner/cfgov-django-setup/blob/master/README.md#cfgov-setup).

But then, I also wonder if it's useful to include links to where this code *should* be used, rather than where it *is* used.